datafusion/sqllogictest/test_files/array.slt

```diff
@@ -1044,6 +1044,20 @@ select make_array(['a','b'], null);
 ----
 [[a, b], ]
 
+## array_sort (aliases: `list_sort`)
+query ???
+select array_sort(make_array(4, 2, 3, 1)), array_sort(make_array(1.0, 4.0, null, 3.0), true, true), array_sort(make_array('a', 'd', 'c', null, 'b'), false, false);
```

Comment:
btw, you can also try column-wise support, pass the test with `table`, you can refer other array_function that run with `table` cases. But it is fine for me to support that in follow on PR